ci/run_tests.sh

```diff
@@ -98,5 +98,36 @@ pushd c_header_only
 "$fpm" build
 popd
 
+pushd profiles_with_c
+"$fpm" build
+"$fpm" run
+popd
+
+pushd program_with_compiler_profiles
+"$fpm" build
+"$fpm" run
+popd
+
+pushd program_with_free_form_in_dot_f
+"$fpm" build
+"$fpm" run
+popd
+
+pushd "program_with_profiles_scope/primary_package"
+"$fpm" build
+"$fpm" run
+popd
+
+pushd "profiles_priorities/main_package"
+rm -rf build
+if [ `uname -s` = "Windows" ]; then
+	"$fpm" build | sed -n 's,^.*\\\([^\\ ]*\.f90 .*\) -J .*$,\1,p' | sort > log.txt
+else
+	"$fpm" build | sed -n 's,^.*/\([^/ ]*\.f90 .*\) -J .*$,\1,p' | sort > log.txt
+fi
```

Comment:
This check is fragile as it relies on the output fpm is producing, which is not guaranteed to be stable.